partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/WhitelistedTests.txt

```diff
@@ -3070,3 +3070,9 @@ run/virtpatmat_switch.scala
 run/t5629b.scala
 run/t6318_primitives.scala
 run/t8764.scala
+
+# Big numerics added
```

Comment:
Ah right, the last blocks are commented with the adaptations we had to make. Then append your additional lines, without comment, at the end of the first gigantic block, i.e., before the first comment, i.e., after line 3026.